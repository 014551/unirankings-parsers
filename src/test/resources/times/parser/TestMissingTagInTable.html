<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>TestEmptyTable</title>
</head>
<body>
<table id="datatable-1"
       class="table rankingsRegWall-processed wur-hash-processed wur-cols-processed wur-pagelen-processed dataTable no-footer rank-only usr-processed table-row-click-processed scores"
       data-module="rankings_table" role="grid" aria-describedby="datatable-1_info">
    <thead>
    <tr role="row">
        <th class="rank sorting_asc" tabindex="0" aria-controls="datatable-1" rowspan="1" colspan="1"
            aria-label="Rank: activate to sort column ascending">Rank
        </th>
        <th class="name namesearch sorting" tabindex="0" aria-controls="datatable-1" rowspan="1" colspan="1"
            aria-label="NameCountry/Region: activate to sort column ascending">Name<br>Country/Region
        </th>
        <th class="scores overall-score sorting" title="The overall score" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="Overall: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">Overall</div>
        </th>
        <th class="scores teaching-score sorting" title="" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="Teaching: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">Teaching</div>
        </th>
        <th class="scores research-score sorting" title="" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="Research: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">Research</div>
        </th>
        <th class="scores citations-score sorting" title="" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="Citations: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">Citations</div>
        </th>
        <th class="scores industry_income-score sorting" title="" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="Industry Income: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">Industry Income</div>
        </th>
        <th class="scores international_outlook-score sorting" title="" data-placement="top" data-toggle="tooltip"
            data-container="body" tabindex="0" aria-controls="datatable-1"
            aria-label="International Outlook: activate to sort column ascending" rowspan="1" colspan="1">
            <div class="rotate">International Outlook</div>
        </th>
    </tr>
    </thead>
    <tbody>
    <tr role="row" class="odd row-1 js-row" data-nid="468">
        <td class="rank sorting_1 sorting_2">1</td>
        <td class=" name namesearch"><a class="ranking-institution-title"
                                        href="/world-university-rankings/university-oxford" data-position="title"
                                        data-mz="">University of Oxford</a>
            <div class="location-courses js-location-courses">
                <div class="location"><span><a href="/location/a4zw0000000GnxQAAS">United Kingdom</a></span></div>
                <a href="/cn/world-university-rankings/university-oxford?utm_medium=thewebsite&amp;utm_campaign=cta-link&amp;utm_source=rankings#institution-courses-pane"
                   class="btn btn-default btn-sm courses-button js-courses-button" data-position="uni-apply" data-mz=""
                   data-original-title="" title=""><span>课程</span></a></div>
        </td>
        <td class=" scores overall-score">94.3</td>
        <!--        Intentionally commented to test the missing tag in the table -->
        <!--        <td class=" scores teaching-score">86.7</td>-->
        <td class=" scores research-score">99.5</td>
        <!--        Intentionally commented to test the missing tag in the table -->
        <!--        <td class=" scores citations-score">99.1</td>-->
        <td class=" scores industry_income-score">63.7</td>
        <td class=" scores international_outlook-score">95.0</td>
    </tr>
</table>
</body>
</html>