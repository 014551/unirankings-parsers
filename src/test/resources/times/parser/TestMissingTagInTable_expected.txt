"0";"1";"University of Oxford";"/world-university-rankings/university-oxford";"United Kingdom";"94.3";;"99.5";;"63.7";"95.0"
